From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
Date: Tue, 31 Oct 2017 21:23:43 +0300
Subject: [ARCHEOLOGY] Add mvebu dev branch

> X-Git-Archeology: > recovered message: > It uses the same kernel as next, but it uses mainline u-boot with DT
> X-Git-Archeology: > recovered message: > overlay support, but without SPI, USB and possibly SATA support
> X-Git-Archeology: - Revision aba730f4880b20f7ba37504a8d1d6bc484f28c54: https://github.com/armbian/build/commit/aba730f4880b20f7ba37504a8d1d6bc484f28c54
> X-Git-Archeology:   Date: Tue, 31 Oct 2017 21:23:43 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Add mvebu dev branch
> X-Git-Archeology:
> X-Git-Archeology: - Revision e71d1560f0429d9ecbc077ac457c6247735e3e9a: https://github.com/armbian/build/commit/e71d1560f0429d9ecbc077ac457c6247735e3e9a
> X-Git-Archeology:   Date: Fri, 23 Nov 2018 15:39:23 +0100
> X-Git-Archeology:   From: aprayoga <aprayoga@users.noreply.github.com>
> X-Git-Archeology:   Subject: Helios4: various updates (#1161)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 3a185c8b8262a3189fd99fedc4350d738e825d0d: https://github.com/armbian/build/commit/3a185c8b8262a3189fd99fedc4350d738e825d0d
> X-Git-Archeology:   Date: Thu, 13 Dec 2018 19:04:47 -0500
> X-Git-Archeology:   From: Thomas McKahan <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: [ rockchip-dev ] Add DT overlay framework
> X-Git-Archeology:
> X-Git-Archeology: - Revision de26797423e22d58ec2882d7032c67f77196ecc5: https://github.com/armbian/build/commit/de26797423e22d58ec2882d7032c67f77196ecc5
> X-Git-Archeology:   Date: Sun, 06 Nov 2022 20:32:46 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Move all legacy u-boot patches under one general legacy folder (#4386)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 97c34489831f2146940f52915428263b7edfcbe1: https://github.com/armbian/build/commit/97c34489831f2146940f52915428263b7edfcbe1
> X-Git-Archeology:   Date: Fri, 24 Mar 2023 23:13:42 +0100
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: put all rockchip 32 bit into uboot/v2022.04 directory
> X-Git-Archeology:
---
 arch/arm/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 111111111111..222222222222 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1908,6 +1908,8 @@ config ARCH_ROCKCHIP
 	select DM_SPI
 	select DM_SPI_FLASH
 	select ENABLE_ARM_SOC_BOOT0_HOOK
+	select OF_LIBFDT
+	select OF_LIBFDT_OVERLAY
 	select OF_CONTROL
 	select SPI
 	select SPL_DM if SPL
-- 
Armbian

